docs/artifacts_test.md

```diff
@@ -118,7 +118,7 @@ SCT_AMI_ID_DB_SCYLLA="<ami you've found>" SCT_REGION_NAME=us-east-1 hydra run-te
 If you want to run against some official ScyllaDB Docker image, you should go to [tag list](https://hub.docker.com/r/scylladb/scylla/tags) and choose some tag name (i.e., `3.3.rc1`)
 
 ```sh
-SCT_SCYLLA_VERSION=<tag you've chose> hydra run-test artifacts_test --backend docker --config test-cases/artifacts/docker.yaml
+SCT_SCYLLA_VERSION="<tag you've chose>" SCT_SCYLLA_REPO="<URL to repo which used to build Docker image>" hydra run-test artifacts_test --backend docker --config test-cases/artifacts/docker.yaml
```

Comment:
The repo file exists in the docker file system `/etc/yum.repos.d/scylla.repo`.
We don't need to assign it by the SCT parameter.

Sometimes, we only know the docker tag, but we don't know the repo url which is used to build the Docker image.

Reply:
This is great. It'll simplify things so much.